wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ProjectHandlerRegistry.java

```diff
@@ -40,8 +39,8 @@ public ProjectHandlerRegistry(Set<ProjectHandler> projectHandlers) {
   public void register(@NotNull ProjectHandler handler) {
     if (handler instanceof CreateProjectHandler) {
       createProjectHandlers.put(handler.getProjectType(), (CreateProjectHandler) handler);
-    } else if (handler instanceof GetItemHandler) {
-      getItemHandlers.put(handler.getProjectType(), (GetItemHandler) handler);
+      //    } else if (handler instanceof GetItemHandler) {
```

Comment:
I guess commented code here and further in the class should be removed?

Reply:
done